eth2_testgen/bls/bls.py

```diff
@@ -0,0 +1,268 @@
+from typing import (  # noqa: F401
```

Comment:
Do we plan to eventually do an import of `bls.py` and other utilities from py-evm rather than a copy paste once we get things more stable on py-evm/trinity side?

Reply:
FYI we plan to [move `trinity/eth2/_utls/bls.py` to py_ecc](https://github.com/ethereum/trinity/issues/137) one day.